airdrop/airdrop-distributor/tests/functions/end_block.rs

```diff
@@ -5,6 +5,8 @@ use crate::utils::{
 
 mod success {
 
+    use fuels::prelude::Bits256;
```

Comment:
Was this meant to be added here or at the top of the file?

Reply:
yep good catch!

Reply:
I think there's another one in a different file just as a heads up